docs/manage/configuration.md

```diff
@@ -74,6 +74,12 @@ You can use the mode selector to switch between **Builder** and **Raw JSON**:
 
 Regardless of the mode you choose, Viam stores the configuration file in [JSON (JavaScript Object Notation)](https://en.wikipedia.org/wiki/JSON).
 
+{{< alert title="Simultaneous config edits" color="caution" >}}
+If you edit a config while someone else edits the same config, the person who saves last will overwrite any other changes.

```

Comment:
```suggestion
If you edit a config while someone else edits the same config, the person who saves last will overwrite any prior changes.
```
(opt. Maybe this clarification could be useful) 
/ past / any past / previous?

Reply:
```suggestion
If you edit a config while someone else edits the same config, the person who saves last will overwrite any prior changes that aren't reflected in the new config.
```